packages/vscode-extension/e2e_tests/tests/signatureHelp.spec.ts

```diff
@@ -61,7 +64,11 @@ suite('Signature help spec', () => {
       // We don't test the active signature, otherwise we would have to modify
       // these tests every time a new function is added to the database
       activeSignature: undefined,
-      signatures: [testData.mockSchema.functionSignatures.abs],
+      signatures: [
+        toSignatureInformation(
+          testData.mockSchema.functions.abs,
+        ) as vscode.SignatureInformation,
```

Comment:
Is this casting needed because of a mismatch in vscode types? I noticed you updates some of the vscode types

Reply:
Yeah, that was a futile attempt to make them work without the casting. It's silly but the types of the language server have sometimes a bit of a mismatch from `@vscode/types`, which other people have reported. I tried updating the package but it doesn't solve it. 

In this case it's cause the documentation can be of type `string | MarkupContent` in one and `string | MarkdownString` in another one.

Reply:
hopefully they'll solve it in a later version, doesn't seem to matter for us at least